packages/skeleton-react/src/components/Tab/Tabs.tsx

```diff
@@ -0,0 +1,216 @@
+import React, { useEffect, useRef, useState } from "react";
+import { TabsProps, TabsListProps, TabsControlProps, TabsControlItemProps, TabsPanelItemProps } from "./types";
+
+// Components ---
+const TabsRoot: React.FC<TabsProps> = ({
+    id = '',
+    // Root
+    base = 'w-full',
+    spaceY = 'space-y-4',
+    classes = '',
+    // Children
+    children
+}) => {
+    return (
+        <div id={id} className={`${base} ${spaceY} ${classes}`} data-testid="tabs">
+            {children}
+        </div>
+    );
+};
+
+const TabsListRoot: React.FC<TabsListProps> = ({
+    // Root
+    base = 'flex',
+    justify = 'justify-start',
+    gap = 'gap-2',
+    border = 'border-b-[1px] border-surface-200-800',
+    classes = '',
+    // Children
+    children
+}) => {
+    return (
+        <div className={`${base} ${gap} ${justify} ${border} ${classes}`} role="tablist">
+            {children}
+		</div>
+    );
+};
+
+const TabsControlRoot: React.FC<TabsControlProps> = ({
+    id = '',
+    name,
+    group,
+    title = '',
+    // A11y
+    label = '',
+    controls = '',
+    // Root
+    base = 'group',
+    active = 'text-surface-950-50 border-surface-950-50',
+    inactive = 'text-surface-600-400 border-transparent',
+    flex = 'flex justify-center items-center',
+    background = '',
+    border = 'border-b-[1px]',
+    text = 'type-scale-3',
+    padding = 'pb-2',
+    rounded = '',
+    gap = 'gap-1',
+    cursor = 'cursor-pointer',
+    classes = '',
+    // Events
+    onclick = () => {},
```

Comment:
@Hugos68 caught a convention issue here. It looks like events differ between Svelte 5 and React:

* Svelte: `onclick` -> all lowercase
* React: `onClick` -> camel-case

Hugo made the suggest that we lean into the framework convention and I think I agree here. We're going to move this to approach for the Accordion and update the contribution docs with a note for this asap.

Make sure to update all events here - click, keydown, etc. Note this change is ONLY needed for the React components. Svelte should remain as is!